deploy/openshift/ocp.sh

```diff
@@ -237,6 +243,18 @@ deploy_che_to_ocp() {
   fi
 }
 
+deployChePluginRegistry() {
+if [ "${DEPLOY_CHE_PLUGIN_REGISTRY}" == "true" ]; then
+  echo "Deploying Che plugin registry..."
+  ${OC_BINARY} new-app -f ${BASE_DIR}/templates/che-plugin-registry.yml \
+             -p IMAGE=${CHE_PLUGIN_REGISTRY_IMAGE} \
+             -p IMAGE_TAG=${CHE_PLUGIN_REGISTRY_IMAGE_TAG}
+  CHE_PLUGIN_REGISTRY_ROUTE=$($OC_BINARY get route/che-plugin-registry --namespace=${CHE_OPENSHIFT_PROJECT} -o=jsonpath={'.spec.host'})
+  echo "Che plugin registry deployment complete. $CHE_PLUGIN_REGISTRY_ROUTE"
+  ${OC_BINARY} set env dc/che CHE_PLUGIN_REGISTRY_URL="http://$CHE_PLUGIN_REGISTRY_ROUTE/plugins/"
```

Comment:
As far as I understand it will invoke che deploy one more time. So, user has to wait until Che becomes ready one more time. Can we avoid this? Like deploy the registry and only then deploy Che with right env variable there?

Reply:
I think we can avoid it. Not sure about do we need to. @garagatyi wdyt?

Reply:
I completely agree with @sleshchenko! 
We can add a variable to Che deployment and set it to the correct value in case registry is needed.

Reply:
I'll take a look.

Reply:
@sleshchenko @garagatyi I've tried to do what you want. But honestly, it required some complex refactoring of ocp.sh and che_deploy.sh that will be thrown away when plugin-registry become a part of che. do you think it's mandatory for this pr?

Reply:
Is not exporting of `CHE_PLUGIN_REGISTRY_URL` enough since deploy script automatically detect envs vars beginning with "CHE_" and passes them to Che deployments?

Reply:
we can do that way, but I would say it's a workaround and prefer to do that over the template. Another issue is that registry deployment requires che to be already deployed. To change that I need to rewrite a lot in che_deploy.sh

Reply:
@sleshchenko I'll try to test your idea.